test/functional/helpers/configureAlloyInstance/index.js

```diff
@@ -1,7 +1,16 @@
 import { ClientFunction } from "testcafe";
 
-const configureAlloyInstance = ClientFunction((instanceName, cfg) =>
-  window[instanceName]("configure", cfg)
-);
+const configureAlloyInstance = ClientFunction((instanceName, cfg) => {
+  let finalInstanceName = instanceName;
+  let finalConfig = cfg;
+
+  // Default instanceName to `alloy`.
+  if (typeof instanceName === "object" && instanceName !== null) {
+    finalInstanceName = "alloy";
+    finalConfig = instanceName;
```

Comment:
Could we just pass an object { instanceName, cfg } here instead to avoid this?

Reply:
I thought about it but didn’t feel like refactoring all the tests 😂